docs/evaluation/how_to_guides/index.md

```diff
@@ -13,6 +13,7 @@ Write evaluations to test and improve your application.
 - [Evaluate an existing experiment](./how_to_guides/evaluation/evaluate_existing_experiment)
 - [Run a pairwise evaluation](./how_to_guides/evaluation/evaluate_pairwise)
 - [Run evals using the API only](./how_to_guides/evaluation/run_evals_api_only)
+- [Use prebuilt LLM-as-a-judge evaluators](./how_to_guides/evaluation/use_prebuilt_llm_evaluator)
```

Comment:
should we be updating the existing how to guide on off the shelf evaluators and replacing with this one?

Reply:
will defer to @agola11 for doc restructuring, but I was under the impression that replacement is the end goal